components/home/Partners.tsx

```diff
@@ -4,13 +4,22 @@ import { PartnersProps } from '@interfaces/home/Partners';
 // library
 import { Fragment } from 'react';
 import Image from 'next/image';
+import { Box, Text } from '@chakra-ui/react'
+
 
 const Partners = ({ partnersImages }: PartnersProps) => {
   return (
     <div className="section-my">
-      <h2 className="mb-16 text-center text-h2 font-bold">
-        Past and Present Partners
-      </h2>
+      <h3 className='absolute m-12 mt-[80px] hidden md:block md:ml-[475px] lg:ml-[710px] xl:ml-[950px]  text-xl text-white max-w-lg '>Past and Present</h3>
+      <div className='w-full flex flex-wrap rounded-[20px] p-8 mb-8 '>
+        <div className='w-full h-[122px] flex flex-wrap bg-[#004F5080] rounded-[20px]'>
+          <img src='/images/home/ourpartners.png' className= 'w-full h-full rounded-[20px]' />
+        </div>
+        <div className='absolute flex m-[70px] mt-[40px] text-4xl text-center sm:text-5xl sm:mt-8 font-bold text-white'>
```

Comment:
Similarly, try to use `mt-10` instead of `mt-[40px]`

Reply:
I have referenced your code for the header instead!